tests/toolbox_image.sh

```diff
@@ -51,6 +113,10 @@ teardown()
         return
     fi
 
+    if [ -r "${STORAGE_DEVICE_FILE}" ]; then
+        unlink "${STORAGE_DEVICE_FILE}"
```

Comment:
could it be the case that it is still mounted? if not, then this is obviously fine

Reply:
Still being mounted would be an error of the setup_storage script then.